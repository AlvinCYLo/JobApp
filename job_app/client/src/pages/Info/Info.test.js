import React from 'react';
import renderer from 'react-test-renderer';
import Info from './Info';

it('renders Info page correct', () => {
  const tree = renderer.create(<Info />).toJSON();
  expect(tree).toMatchSnapshot();
});
