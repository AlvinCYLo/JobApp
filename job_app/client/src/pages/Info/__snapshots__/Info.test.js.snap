// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Info page correct 1`] = `
<div>
  INFO
</div>
`;
